genestack_client/settings/genestack_user.py

```diff
@@ -17,9 +17,9 @@ def _get_server_url(host):
 
 class User(object):
     """
-    A class storing information about the server's URL, the user login, password and alias.
+    A class storing information about the server's URL, the user login, password, token and alias.
```

Comment:
On this comment i thought about possibility to set up both — login/password and token. Is it possible now? Maybe it would be better to write "URL, alias, and user/password pair or API token".

Reply:
Yes it is possible, but I will change message in the way you suggested.